integrations/llama_index/src/ucai_llamaindex/toolkit.py

```diff
@@ -41,6 +42,13 @@ def __init__(
         super().__init__(*args, fn=fn, metadata=metadata, **kwargs)
         self.client_config = client_config
 
+    def __repr__(self) -> str:
+        return (
+            "UnityCatalogTool("
+            + ", ".join([f"{k}={v}" for k, v in asdict(self.metadata).items() if v])
```

Comment:
```suggestion
            + ", ".join(f"{k}={v}" for k, v in asdict(self.metadata).items() if v)
```

to skip temporary list creation.